packages/compass-export-to-language/src/stores/store.js

```diff
@@ -23,6 +24,33 @@ export const setCopyToClipboardFn = (store, fn) => {
   store.dispatch(copyToClipboardFnChanged(fn));
 };
 
+// TODO: replace this with state coming from the right layer.
+// this kind of information should not be derived
+// from dataService as it operates on a lower level,
+// as a consequence here we have to remove the `appName` that only
+// the dataService should be using.
+function getCurrentlyConnectedUri(dataService) {
+  let connectionStringUrl;
+
+  try {
+    connectionStringUrl = new ConnectionString(
+      dataService.getConnectionOptions().connectionString
+    );
+  } catch (e) {
+    return '<uri>';
```

Comment:
sorry for the "diaper" here, should help with the integration in DE and be innocent in Compass since we have tests that would catch issues. Hopefully will go away soon